pom.xml

```diff
@@ -888,8 +888,24 @@
             <url>${maven.repository.url}</url>
             <layout>default</layout>
         </repository>
+        <!-- FIXME WFLY-19560 temp, must be removed before merge -->
+        <repository>
+            <id>jakarta-staging</id>
+            <name>Jakarta Staging</name>
+            <url>https://jakarta.oss.sonatype.org/content/repositories/staging/</url>
+            <layout>default</layout>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+                <updatePolicy>never</updatePolicy>
+            </snapshots>
+        </repository>
```

Comment:
We can't merge this as-is, but I understand why it's there. I've sent an email to the glassfish-dev mailing list asking if there can be a release of concurro.